FhirToDataLake/src/Microsoft.Health.Fhir.Synapse.DataClient/Api/Dicom/DicomApiConstants.cs

```diff
@@ -0,0 +1,72 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using Microsoft.Health.Fhir.Synapse.Common;
+
+namespace Microsoft.Health.Fhir.Synapse.DataClient.Api.Dicom
+{
+    public class DicomApiConstants
+    {
```

Comment:
nit: add a link of the dicom api as comment?

Reply:
Added, thanks!